manifests/w/w1hkj/fldigi/4.2.11/w1hkj.fldigi.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created with komac v2.14.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
+
+PackageIdentifier: w1hkj.fldigi
+PackageVersion: 4.2.11
+PackageLocale: en-US
+Publisher: W1HKJ Software
```

```suggestion
Publisher: Fldigi developers
```
This should match what is written to control panel, or `AppsAndFeaturesEntries` should be added to the installer manifest